bridge/framework/django.py

```diff
@@ -33,17 +34,28 @@ def configure_postgres(self, platform: Platform) -> None:
             }
         }
 
+    def configure_redis(self, config: RedisConfig) -> None:
+        self.framework_locals["CACHES"] = {
+            "default": {
+                "BACKEND": "django.core.cache.backends.redis.RedisCache",
+                "LOCATION": "redis://127.0.0.1:6379",
```

Comment:
Should this be reading from the config argument/take in information about the host/port?

Also might be better to adopt the new pattern taking the platform as an argument and building the config in the 'platform' package.